pkg/sql/export.go

```diff
@@ -196,6 +200,10 @@ func buildExportPlanningInfo(
 	exprEval := planner.ExprEvaluator("EXPORT")
 	treeOptions := make(tree.KVOptions, len(options))
 	for i, o := range options {
+		if o.Value.(*tree.CastExpr).Expr == tree.DNull {
```

Comment:
is this type assertion always valid? if not, do an `, ok ` check. i dont know enough about this code to say one way or the other.